core/src/components/calendar/calendar.tsx

```diff
@@ -0,0 +1,555 @@
+import { Component, ComponentInterface, Element, Event, EventEmitter, Host, Prop, State, Watch, h } from '@stencil/core';
+import { addMonths, addYears, eachDayOfInterval, eachYearOfInterval, endOfMonth, endOfWeek, format, startOfMonth, startOfWeek } from 'date-fns';
+
+import { getIonMode } from '../../global/ionic-global';
+import { createAnimation } from '../../utils/animation/animation';
+import { Gesture, GestureDetail } from '../../utils/gesture';
+
+import { DAY_SHORT_NAMES } from './calendar-util';
+
+/**
+ * @virtualProp {'ios' | 'md'} mode - The mode determines which platform styles to use.
+ */
+@Component({
+  tag: 'ion-calendar',
+  shadow: true,
+  styleUrls: {
+    ios: 'calendar.ios.scss',
+    md: 'calendar.md.scss'
+  }
+})
+export class Calendar implements ComponentInterface {
+  private monthsWrapper!: HTMLElement;
+  private today: Date = new Date();
+  private gesture?: Gesture;
+  private dragging = false;
+
+  @Element()
```

Comment:
Can you make sure all of these are on one line please? We follow this guide: https://stenciljs.com/docs/style-guide